actions.py

```diff
@@ -0,0 +1,46 @@
+students = []
+
+def add_student():
+    try:
```

Comment:
Add validation of scores to each individual score

Reply:
Validation has been added to the program: it will not allow scores greater than 100 or letters.

![changes](https://github.com/JorgeA/DUAD/assets/4186222/4caf79eb-dddc-4733-a191-bddcff2e1e75)

Reply:
"I finally fixed the issue with the validation, so the user will not need to start over with the student registration if they mistakenly type a letter in the score field.
![image](https://github.com/JorgeA/DUAD/assets/4186222/2e4489c4-f72d-4e73-a09e-bd985930b3b5)